### DISCUSSION TITLE
Eraser for canvas (Scribble model)

### DISCUSSION THREAD

**Discussion by sashasubbbb** [Mikubill/sd-webui-controlnet#231] [Ideas]
Is it possible to have an eraser for scribble model on canvas? Or just implement color on brushes just like in inpaint mode, so we can imitate an eraser by drawing with white brush. Right now the only thing is to press undo, but it's not ideal.

**Comment by Alchete** [comment]
The image features are limited by Gradio, which is the UI of choice for A1111.  So there can't be an eraser until Gradio implements one.

**Reply by sashasubbbb** [reply]
But SD gradio webui had color-sketch tool (now it’s called inpaint sketch in img2img tab). Can’t we use white color as an eraser for sketching?
